### PR TITLE
Add mergify team-members

### PR DESCRIPTION
**Pull request by Nebukadneza**
## What type of PR?

enhancement

## What does this PR do?

Sync the newly added team-members to the mergify team-regex so that they will benefit from the "team needs only one approve" rule

## Prerequistes
- [x] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/guide.html#changelog) entry file.

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ pull_request_rules:
         message: |
           Thanks for submitting this pull request.
           Bors-ng will now build test images. When it succeeds, we will continue to review and test your PR.
-
```
Comment (lub) [inline .mergify.yml:14]: Aren't these indentations required by the yaml spec? Afaik | continues until indentation changes.

Comment (Diman0) [reply .mergify.yml:14]: Yes. These indentations are required to continue the multiline string.

File: .mergify.yml [hunk 2]
```diff
@@ -11,9 +11,9 @@ pull_request_rules:
         message: |
           Thanks for submitting this pull request.
           Bors-ng will now build test images. When it succeeds, we will continue to review and test your PR.
-          
+
           bors try
-
```
Comment (Diman0) [inline .mergify.yml:16]: These indentations are required to continue the multiline string. Please use space characters for the indendation.

**Review by Diman0 — CHANGES_REQUESTED** [review]
so messages from lub and me. You removed indendations from the yaml file which are required to continue the multiline text.

File: .mergify.yml (hunk 2 above)
Comment (Nebukadneza) [reply .mergify.yml]: Argh, sorry for not having paid attention. Thanks for the pointer!

**Review by Diman0 — APPROVED** [review]
Looks good to me

**Comment by Nebukadneza** [comment]
In accordance with @Diman0 we believe this is now fixed, and since mergify won’t do it’s job on PRs containing mergify config, let’s do this by hand:

bors r+

**Pull request merged**